src/posthog-core.js

```diff
@@ -139,8 +139,6 @@ var create_mplib = function (token, config, name) {
     instance['people']._init(instance)
 
     instance.featureFlags = new PostHogFeatureFlags(instance)
-    // This key is deprecated
-    instance.feature_flags = instance.featureFlags
```

Comment:
Removing this seems honestly like the riskiest thing here. Up to you but calling this out.

Reply:
yeah I thought that too. It's been deprecated for almost a year, but I guess we never issued a warning for this, so might make sense to revert